config/configtls/configtls.go

```diff
@@ -41,11 +41,13 @@ type TLSSetting struct {
 	KeyFile string `mapstructure:"key_file"`
 
 	// MinVersion sets the minimum TLS version that is acceptable.
-	// If not set, TLS 1.0 is used. (optional)
+	// If not set, refer to crypto/tls for defaults. (optional)
+	// Which is TLS 1.2 for clients and TLS 1.0 for servers
```

Comment:
I'd remove the last line here, otherwise this will eventually get out of sync if/when crypto/tls changes its defaults

Reply:
Altered